lib/copy.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs-extra');
+const path = require('path');
+const oryx = require('@spryker/oryx');
+const settings = require('./settings');
+
+const copyAssets = async () => {
+    const backCompatibilityPublicDir = path.resolve(path.join('public/Zed', settings.paths.publicPath));
+
+    try {
+        oryx.log.task('copy');
```

Comment:
Suggest to add more information about this step. Something like:
```suggestion
        oryx.log.task('Copy assets for backward compatibility after renaming assets folder');
```